### PR TITLE
Check response data type before appending

### PR DESCRIPTION
**Pull request by mycarrysun**
My implementation of responses sometimes has integers as the response content. This will of course produce an error trying to access an index of an integer like an array.

I have added a check to make sure the response data is an array before trying to attach the debug property to the response.

Also: I have added a small convenience for seeing the total time of all the queries in the query collection.

### CODE REVIEW

File: tests/DebuggerTest.php [hunk 1]
```diff
@@ -199,4 +199,23 @@ public function it_can_show_cache_events()
                 ],
             ]);
     }
+
```
Comment (mlanin) [inline tests/DebuggerTest.php]: Please, add `/** @test */` annotation.
Without it, phpunit will skip it

**Comment by mycarrysun** [comment]
@mlanin added :)

**Comment by mlanin** [comment]
Tests are failing now :(

**Comment by mycarrysun** [comment]
@mlanin looks like for some reason the `$this->json()` method is returning the standard `Illuminate\Http\Response` type instead of the `TestResponse`, which makes sense why it can't find the `assertJsonMissingExact()` method.

Did I do anything wrong in the return type?

```php
/** @test */
public function is_does_not_add_debug_if_response_is_integer()
{
    $this->app['router']->get('foo', function () {
        lad('test if integer');

        return response()->json(1234);
    });

    $this->json('get', '/foo')
         ->assertStatus(200)
         ->assertJsonMissingExact([
            'debug' => [
                'dump' => [
                    'test if integer',
                ]
            ]
         ]);
}
```

**Comment by mlanin** [comment]
Tests are failing for php 5.6 version. The highest laravel version that supports this is 5.4.36. But this version doesn't have `assertJsonMissingExact` method :(

**Comment by mycarrysun** [comment]
Can we include this version requirement of laravel in composer.json then? The copy I have installed v5.7.28.

**Comment by mlanin** [comment]
But this package supports 5.4 version. Your PR is really good, please, adjust your test and I'll merge it asap :)

**Comment by mycarrysun** [comment]
> But this package supports 5.4 version. Your PR is really good, please, adjust your test and I'll merge it asap :)

I've updated the `composer.json` to use version of packages that are compatible with your requirements so composer doesn't try to install a newer version. I also added requirements for `sqlite` since it was missing from my machine and the tests were failing (Ubuntu 19).

File: composer.json [hunk 2]
```diff
@@ -11,12 +11,14 @@
         }
     ],
     "require": {
+        "ext-sqlite3": "*",
+        "ext-pdo_sqlite": "*",
```
Comment (mlanin) [inline composer.json]: ext-sqlite3 is not a dependency for this package

Comment (mycarrysun) [reply composer.json]: Is it just pdo_sqlite then?

Comment (mlanin) [reply composer.json]: `pdo_sqlite` as well. This package doesn't use sqlite

Comment (mycarrysun) [reply composer.json]: Yes it does - that's why I added it. The unit tests use a sqlite database connection.

Comment (mycarrysun) [reply composer.json]: Maybe it should go in require-dev?

**Comment by mycarrysun** [comment]
Closing from lack of activity from the maintainer. Feel free to reopen if you want it merged.

**Pull request closed without merging**